plugins/loading/src/index.ts

```diff
@@ -33,20 +51,32 @@ const loadingPlugin = (config: LoadingConfig = {}): any => {
   if (config.whitelist && config.blacklist) {
     throw new Error('loading plugin config cannot have both a whitelist & a blacklist')
   }
+}
+
+export default (config: LoadingConfig = {}): any => {
+  validateConfig(config)
 
   const loadingModelName = config.name || 'loading'
+
+  const converter =
```

Comment:
I'm pretty happy with this solution, it keeps a "background" counter called `cntState` of dispatched effects, but converts it onto its public state as either a number or boolean.